controllers/vgmanager_daemonset.go

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	lvmv1alpha1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	hostContainerPropagation  = corev1.MountPropagationHostToContainer
+	directoryHostPath         = corev1.HostPathDirectory
+	HostPathDirectoryOrCreate = corev1.HostPathDirectoryOrCreate
+
+	LVMdVolName   = "lvmd-conf"
+	UdevVolName   = "run-udev"
+	DevDirVolName = "device-dir"
+	SysVolName    = "sys"
+
+	LVMdDir      = "/mnt/lvmd/"
+	LVMdConfPath = "/mnt/lvmd/lvmd.conf"
```

Comment:
- Did I miss anything or is this not being used in this PR?

Reply:
Its not being used. I've removed it.